src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 410px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Use `padding: 32px 16px 16px;` instead